src/components/progress-indicator/bl-progress-indicator.ts

```diff
@@ -0,0 +1,47 @@
+import { CSSResultGroup, html, LitElement, TemplateResult } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import style from './bl-progress-indicator.css';
+
+export type ProgressIndicatorSize = 'small' | 'medium' | 'large';
+
+/**
+ * @tag bl-progress-indicator
+ * @summary Baklava Progress Indicator component
+ */
+
+@customElement('bl-progress-indicator')
+export default class BlProgress extends LitElement {
+  static get styles(): CSSResultGroup {
+    return style;
+  }
+
+  @property({ type: String })
+  size: ProgressIndicatorSize = 'medium';
+
+  @property({ type: String })
+  max = '100';
+
+  @property({ type: String })
+  value = '0';
```

Comment:
Use `number` type for those properties (as we did for `max` and `min` properties in `bl-input`)

Reply:
Actually, I used the number first but if I use the number i have to set property like this:

```
   this.style.setProperty('--bl-progress-indicator-max', this.max.toString());
   this.style.setProperty('--bl-progress-indicator-value', this.value.toString());
```
I didn't want to use `toString()` method and we are not calculating in js with these values. So I thought it can be` string`.

Reply:
You could use `${this.max}`.